packages/customize-widgets/src/components/header/index.js

```diff
@@ -45,4 +45,4 @@ function Header( { inserter, isInserterOpened, setIsInserterOpened } ) {
 	);
 }
 
-export default Header;
+export default memo( Header );
```

Comment:
Memoing this allow us to prevent re-render it every time we update a block.

Reply:
We don't need this now as the `<Header>` component has been moved outside of the provider.